src/Components/Publishing/Display/Canvas/CanvasContainer.tsx

```diff
@@ -89,7 +89,7 @@ export class CanvasContainerComponent extends React.Component<CanvasContainerPro
       const slideshowProps = { unit, campaign, disclaimer, containerWidth: size.width }
 
       return (
-        <CanvasSlideshow {...slideshowProps}>
+        <CanvasSlideshow {...slideshowProps} renderTitleCard>
```

Comment:
This opens up the space for the mobile slideshow stuff to show but it's still hidden for reasons I can't quite figure out.